example/glue/ConfigurationDelegate.hpp

```diff
@@ -175,6 +175,16 @@ class MM_ConfigurationDelegate
 	 */
 	MM_GCPolicy getGCPolicy() { return _gcPolicy; }
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	/**
+	 * If the checkpointgcthread is specified by the user, it is verified
```

Comment:
To avoid confusion and for clarity: checkpointgcthread -> checkpointGCthreadCount, which is the correct variable name.
```suggestion
	 * If checkpointGCthreadCount is specified by the user, it is verified
```